.azure-pipelines/azure-pipelines-job-groups.yml

```diff
@@ -16,20 +19,25 @@ parameters:
   default: ''
 - name: 'timeoutInMinutes'
   type: 'number'
-  default: 1440
+  default: 600
+- name: 'pool'
+  type: string
+  default: 'sonicbld'
 - name: 'jobFilters'
   type: object
   default: ''
 
 jobs:
 - ${{ each jobGroup in parameters.jobGroups }}:
-  - ${{ if or(eq(parameters.jobFilters, ''), containsValue(parameters.jobFilters, jobGroup.name)) }}:
+  - ${{ if or(eq(parameters.jobFilters, ''), containsValue(parameters.jobFilters, jobGroup.name), endswith(variables['Build.DefinitionName'], format('.{0}{1}', jobGroup.name, jobGroup.extName))) }}:
     - job: ${{ replace(format('{0}{1}', jobGroup.name, jobGroup.extName), '-', '_') }}
       ${{ each pair in jobGroup }}:
         ${{ if not(in(pair.key, 'job', 'name', 'extName', 'variables', 'steps', 'script', 'scriptEnv')) }}:
           ${{ pair.key }}: ${{ pair.value }}
       ${{ if eq(jobGroup.timeoutInMinutes, '') }}:
         timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+      ${{ if eq(jobGroup.pool, '') }}:
```

Comment:
Can be removed, since it has been implemented in line 36.

Reply:
fixed

---
In reply to: [606142279](https://github.com/Azure/sonic-buildimage/pull/7198#discussion_r606142279) [](ancestors = 606142279)